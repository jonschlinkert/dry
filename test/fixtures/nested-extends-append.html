{% extends "nested-extends-append" %}

{% block append="one" %}
one: "nested-extends-append.html"
{% endblock %}

{% block "two" %}
two: "nested-extends-append.html"
{% endblock %}

{% block "five" %}
five: "nested-extends-append.html"
{% endblock %}

{% block "six" %}
six: "nested-extends-append.html"
{% endblock %}

{% block append="seven" %}
seven: "nested-extends-append.html"
{% endblock %}