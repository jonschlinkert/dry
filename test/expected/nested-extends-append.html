
one: "blocks/parent-append.html"
one: "nested-extends-append.html"

two: "nested-extends-append.html"

three: "blocks/parent-append.html"
three: "blocks/nested-extends-append.html"

four: "blocks/parent-append.html"
four: "blocks/nested-extends-append.html"

five: "nested-extends-append.html"

six: "nested-extends-append.html"

seven: "blocks/parent-append.html"
seven: "blocks/nested-extends-append.html"
seven: "nested-extends-append.html"